mkosi/kmod.py

```diff
@@ -189,6 +194,7 @@ def gen_required_kernel_modules(
             },
             mods,
             firmware,
+            links,
```

Comment:
We should make sure the parent directories of these exist as well so please modify the first entry of the chain as well to make the parent directories for all these as well.